Compiler/test/inference.jl

```diff
@@ -823,7 +825,7 @@ end
 
 # Issue 19641
 foo19641() = let a = 1.0
-    Compiler.return_type(x -> x + a, Tuple{Float64})
+    Base.Compiler.return_type(x -> x + a, Tuple{Float64})
```

Comment:
Let's standardize on `Base._return_type` for all of these and what that actually ends up referring to we can update as necessary.